docs/container.md

```diff
@@ -63,12 +64,12 @@ class Config extends ContainerConfig
     {
         $adr = $di->get('radar/adr:adr');
 
+        $adr->middle('Relay\Middleware\ResponseSender');
```

Comment:
(/me furrows brow) I can't tell, within this exchange, what's going on. Better to put together a full example, and ask based on that.